Numbers/StaticConstructorOnGameStart.cs

```diff
@@ -90,9 +92,34 @@ private static void PopulatePresets()
             combatPreset.AddRange(DefDatabase<PawnTableDef>.GetNamed("Numbers_CombatPreset").columns);
             workTabPlusPreset.AddRange(DefDatabase<PawnTableDef>.GetNamed("Numbers_WorkTabPlusPreset").columns);
             colonistNeedsPreset.AddRange(DefDatabase<PawnTableDef>.GetNamed("Numbers_ColonistNeedsPreset").columns);
+            PopulatePsycastingPreset();
             PopulateMedicalPreset();
         }
 
+        private static void PopulatePsycastingPreset()
+        {
+            psycastingPreset.Add(DefDatabase<PawnColumnDef>.GetNamed("Label"));
+            psycastingPreset.Add(DefDatabase<PawnColumnDef>.GetNamed("Numbers_PsylinkLevel"));
+            psycastingPreset.Add(DefDatabase<PawnColumnDef>.GetNamed("Numbers_Psyfocus"));
+            psycastingPreset.Add(DefDatabase<PawnColumnDef>.GetNamed("Numbers_Entropy"));
+            List<PawnColumnDef> abilities = DefDatabase<PawnColumnDef>.AllDefsListForReading.Where(pcd => pcd.Ext()?.ability != null).ToList();
```

Comment:
![errorOnInit](https://user-images.githubusercontent.com/7416299/101870434-9862d080-3b4f-11eb-9517-9090540726e2.jpg)

Reply:
Yeah I personally don't use any mods that would like, add abilities or anything, but I just naturally assumed those mods exist, and so I wanted to do it all dynamically. We could definitely do it explicitly though.

Reply:
481c8c8f7ec8d83d72d0533d78ea95364d49f92f - Add Optional Param to Ext() to Suppress Error